src/m_menu.c

```diff
@@ -3429,6 +3429,10 @@ enum {
   auto1_flash,
 };
 
+static const char *overlay_strings[] = {
```

Comment:
This is the exact same string array as `menu_background_strings[]`. Please reuse.